src/alertconfiguration.cc

```diff
@@ -196,12 +196,21 @@ int AlertConfiguration::
         log_error ("nothing created, lua error");
         return -5;
     }
+
+    // PQSWMBT-3723, don't instanciate sensor temp./humidity rules directly
+    if ((temp_rule->name().find("humidity.default@sensor-") == 0) // starts with...
+        || (temp_rule->name().find("temperature.default@sensor-") == 0))
+    {
+        log_debug("rule instanciation rejected (%s)", temp_rule->name().c_str());
+        return -100;
+    }
```

Comment:
Not a fan of hard-coding logic inside the agent instead of letting the template carry that information. Couldn't you check that the template doesn't have a `CAT_NOINSTANCIATE` tag instead?

Reply:
I took a look at it ... but the CAT property  of the rules seems to not be managed by the service (+CAT values are not enumerated).

Reply:
and about hard-coding, i follow the existing design.